### PR TITLE
handleMouseMove edit

### PR DESCRIPTION
**Pull request by vitbokisch**
Now should be working correctly calculating image coordinates for all angles, not only for 0, 90, 180 and 270 degrees.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -482,34 +481,13 @@ class AvatarEditor extends React.Component {
         y: lastY
       } = this.getCroppingRect()
 
-      lastX *= width
```
Comment (mtlewis) [inline src/index.js]: Just pulled the branch and panning is currently broken due to the removal of 485-6 :)

**Comment by mtlewis** [comment]
Awesome! Just one tiny tweak necessary to get it working.  +1 for this change :)

**Comment by vitbokisch** [comment]
I'm sorry I overlooked it. I merged the latest version to my fork and fixed that issue. Now it should be working! I'm glad you like this  ;)

**Comment by mtlewis** [comment]
Looks like there are commits from master intermingled with the changes in this branch.  This will lead to multiple copies of commits like "Enable source maps in demo" if this PR is merged as-is.  I'd suggest to fix it that you run `git pull --rebase origin master` to fetch origin and rebase your work on top of latest master, and then force push your branch to replace the interleaved commits with the cleaned up ones (with the usual caveats about force pushing ;)).

Incidentally I've found that working on changes in the master branch of a fork creates a lot of unnecessary mental overhead, because you're creating a PR from master -> master.  When I do work like this I usually create a feature branch in my fork.

**Comment by vitbokisch** [comment]
Thank you for your advices I really appreciate it. I decided to delete my forked repo and do it from scratch again as it small change in the code. I'm going to create a new pull request and close this one.

**Pull request closed without merging**